machine.go

```diff
@@ -5,12 +5,11 @@ package gomaasapi
 
 import (
 	"fmt"
-	"net/http"
-	"net/url"
-
 	"github.com/juju/errors"
 	"github.com/juju/schema"
 	"github.com/juju/version"
+	"net/http"
```

Comment:
Please move thee imports up to the first block containing those from the std library

Reply:
done

Reply:
The PR still shows these imports grouped into the wrong block?